profiler/lib.profiler.common/src/org/netbeans/lib/profiler/common/integration/IntegrationUtils.java

```diff
@@ -60,7 +60,6 @@
     public static final String PLATFORM_LINUX_CVM = messages.getString("IntegrationUtils_PlatformLinuxCvm"); // NOI18N
     public static final String PLATFORM_SOLARIS_INTEL_OS = messages.getString("IntegrationUtils_PlatformSolarisIntelOs"); // NOI18N
     public static final String PLATFORM_SOLARIS_AMD64_OS = messages.getString("IntegrationUtils_PlatformSolarisAmd64Os"); // NOI18N
-    public static final String PLATFORM_SOLARIS_SPARC_OS = messages.getString("IntegrationUtils_PlatformSolarisSparcOs"); // NOI18N
```

Comment:
The change in the sig file is a result from the change in this file - the string `PLATFORM_SOLARIS_SPARC_OS` needs to be retained.